### DISCUSSION TITLE
Disable global ban broadcast

### DISCUSSION THREAD

**Discussion by transcental** [EssentialsX/Essentials#5778] [Q&A] [answered]
Hi!
When using `/ban` the ban is broadcast to all server members. Is it possible to disable this?
I can't see a setting for it in `config.yml` but would be surprised if it wasn't possible.

**Comment by ImDarkLaw** [comment, marked as answer]
EssentialsX doesn't assign any of its permissions by default, including `essentials.ban.notify`, which is the permission needed for ban punishment notifications. Double-check that you haven't accidentally assigned this permission to your default group or to a group that all server members inherit permissions from.

*Worth to note that server operators, or users that inherit wildcards (`*`, or `essentials.*`), will also inherit the mentioned permission.*

**Reply by transcental** [reply]
Ah, I'm surprised that we made that mistake. That was the issue, thank you!
